test/powershell/Modules/Microsoft.PowerShell.Core/Enter-PSHostProcess.Tests.ps1

```diff
@@ -132,6 +132,9 @@ Describe "Enter-PSHostProcess tests" -Tag Feature {
                 $npInfo = [System.Management.Automation.Runspaces.NamedPipeConnectionInfo]::new($pwshId)
```

Comment:
@TylerLeonhardt, your last commit had 1 failures in `PowerShell-CI-linux`
Enter-PSHostProcess tests.By Process Id.Can enter using NamedPipeConnectionInfo
```powershell
Expected '[System.Management.Automation.Runspaces.RunspaceState]::Opened', but got Opened.
at <ScriptBlock>, /home/vsts/work/1/s/test/powershell/Modules/Microsoft.PowerShell.Core/Enter-PSHostProcess.Tests.ps1: line 136
136:                     Should -Be [System.Management.Automation.Runspaces.RunspaceState]::Opened -Because $rs.RunspaceStateInfo.Reason
```